document/core/util/macros.def

```diff
@@ -894,15 +894,17 @@
 
 .. |ok| mathdef:: \mathrel{\mbox{ok}}
 .. |const| mathdef:: \xref{valid/instructions}{valid-constant}{\mathrel{\mbox{const}}}
+.. |catch| mathdef:: \xref{valid/conventions}{valid-labelkind}{\mathrel{\mbox{catch}}}
 
+.. |labelkind| mathdef:: \xref{valid/conventions}{valid-labelkind}{\mathrel{\X{labelkind}}}
```

Comment:
Similarly, this is a non-terminal, be typeset with `\X{}`, and should go to the Types section.

Reply:
Did you meant it should be differently typeset? It is already typeset with `\X{}`.